SU2_CFD/include/drivers/CDiscAdjMultizoneDriver.hpp

```diff
@@ -0,0 +1,183 @@
+/*!
+ * \class CDiscAdjMultizoneDriver.hpp
+ * \brief Class for driving adjoint multi-zone problems.
+ * \author O. Burghardt, T. Albring, R. Sanchez
+ * \version 6.2.0 "Falcon"
+ *
+ * The current SU2 release has been coordinated by the
+ * SU2 International Developers Society <www.su2devsociety.org>
+ * with selected contributions from the open-source community.
+ *
+ * The main research teams contributing to the current release are:
+ *  - Prof. Juan J. Alonso's group at Stanford University.
+ *  - Prof. Piero Colonna's group at Delft University of Technology.
+ *  - Prof. Nicolas R. Gauger's group at Kaiserslautern University of Technology.
+ *  - Prof. Alberto Guardone's group at Polytechnic University of Milan.
+ *  - Prof. Rafael Palacios' group at Imperial College London.
+ *  - Prof. Vincent Terrapon's group at the University of Liege.
+ *  - Prof. Edwin van der Weide's group at the University of Twente.
+ *  - Lab. of New Concepts in Aeronautics at Tech. Institute of Aeronautics.
+ *
+ * Copyright 2012-2019, Francisco D. Palacios, Thomas D. Economon,
+ *                      Tim Albring, and the SU2 contributors.
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include "CMultizoneDriver.hpp"
+
+class CDiscAdjMultizoneDriver : public CMultizoneDriver {
+
+protected:
+
+  /*!
+   * \brief Kinds of recordings (three different ones).
+   */
+  enum ENUM_TAPE {
```

Comment:
please consider using `enum class` if you don't need to rely on implicit conversion to integer type

Reply:
Thanks for the hint!
I changed it for the one where I don't want this conversion, it's `enum class Kind_Tape` now. For the other one i need it.. so should one try `enum class Tape_Positions : int { ... } ` then?